bigdecimal.gemspec

```diff
@@ -37,11 +37,4 @@ Gem::Specification.new do |s|
   ]
 
   s.required_ruby_version = Gem::Requirement.new(">= 2.4.0")
-
-  s.add_development_dependency "benchmark_driver"
-  s.add_development_dependency "fiddle"
-  s.add_development_dependency "rake", ">= 12.3.3"
-  s.add_development_dependency "rake-compiler", ">= 0.9"
-  s.add_development_dependency "minitest", "< 5.0.0"
-  s.add_development_dependency "irb"
```

Comment:
Why these parts are moved to Gemfile?

Reply:
Bundler already removed these part with new gem template. Because `add_development_dependency` is less feature than Gemfile like `git`, `group`, `ref` or other specifying function.

Reply:
OK, I understood the current situation of the dependency specification.  Thanks.